psalm-baseline.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="3.9.3@2e4154d76e24d1b4e59e6cc2bebef7790cb9e550">
+  <file src="src/Roave/SecurityAdvisories/Advisory.php">
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>$versionConstraints</code>
+      <code>list&lt;VersionConstraint&gt;</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromFriendsOfPhp.php">
+    <MixedArgument occurrences="1">
+      <code>Yaml::parse(file_get_contents($filePath), Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE)</code>
+    </MixedArgument>
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>SplFileInfo[]</code>
```

Comment:
What's going on here? Broken stubs?

Reply:
I think Psalm doesn't have generics stubs for a lot of Spl Iterators. Searching for CallbackFilterIterator in Psalm repo for example doesn't return anything useful

Reply:
Alright, so that can likely be improved by adding `@extends iterable<SplFileInfo>` or such in upstream at a later point in time.

Reply:
I think so, If I find some time, I'll ask muglug for advice on what should be done. If it's easy, I could do it.